config/production.yaml.example

```diff
@@ -33,6 +33,18 @@ redis:
   auth: null
   db: 0
 
+ldap:
+  enable: true
+  # Disallow non-ldap users (it also disables root!)
+  ldap_only: true
```

Comment:
I'm not sure it's a good idea. What if instead we have something like:

```
auth:
  local:
    enabled: boolean
  ldap:
    enabled: boolean
    ...
```

Reply:
After some thought, I don’t know if it is a good idea to support both, apart from a transitionning period: supporting both local and "remote" (be it ldap or something else) user will most probably provoke clashes in usernames at some point.

So my thought would be to have ldap_only to true, and warn the user that ldap_only: false should be used only during transition to ldap

Reply:
Yes, it was my second concern:

"What do we do when someone registers on the instance with a username that already exists in LDAP, but not in our local database (because the user has not yet connected for example)"

When someone registers, we could check in LDAP if the username already exists. Because even if there is not much sense to have LDAP and local auth enabled, we'll have this kind of use case with other authentication methods.

Anyway, please use the `auth:` parent key. We can then throw errors/warnings at startup if we decide we don't support LDAP & local authentication.